ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -192,14 +206,19 @@ export default defineComponent({
       setMhrTransferCurrentHomeOwnerGroups(currentOwnerGroups)
     }
 
+    const cancel = async (): Promise<void> => {
+      if (hasUnsavedChanges.value === true) localState.showCancelDialog = true
+      else goToDash()
+    }
```

Comment:
Could this check be included in goToDash vs a separate method?

Reply:
I checked onMounted() and also goToReview() where goToDash is being called and if hasUnsavedChanges.value is true than the dialog will display.

Reply:
You are creating a new function that does the very same thing the existing one does but with a condition that could just intercept in our `goToDash` funtion.

Reply:
Made the change and put the owner logic in goToDash.. Works...

Reply:
Sounds good, thanks!